tchannel/format.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import (
```

Comment:
And I mean the most common use case doesn't involve importing any of these modules directly anyway.